networkx/algorithms/hierarchy.py

```diff
@@ -17,8 +18,8 @@ def flow_hierarchy(G, weight=None):
     G : DiGraph or MultiDiGraph
        A directed graph
 
-    weight : key,optional (default=None)
-       Attribute to use for node weights. If None the weight defaults to 1.
+    weight : string, optional (default=None)
+       Attribute to use for edge weights. If None the weight defaults to 1.
```

Comment:
The docstring mistake was uncovered by how we now test the info from `@nx._dispatch`.